admin/includes/functions/general.php

```diff
@@ -763,3 +763,9 @@ function zen_get_orders_status_pulldown_array()
 
     return $orders_status_array;
 }
+
+function zen_get_customer_email_from_id($cid) { 
+   global $db;
+   $query = $db->Execute("SELECT customers_email_address FROM " . TABLE_CUSTOMERS . " WHERE customers_id = " . (int)$cid); 
+   return $query->fields['customers_email_address']; 
```

Comment:
Shouldn't this return an empty string if, for whatever reason, that customers' ID isn't valid?